src/redux/books/books.js

```diff
@@ -13,12 +15,27 @@ export const removeBook = (payload) => ({
   payload,
 });
 
+export const loadBooks = () => (dispatch) => {
+  getBooks().then((books) => {
+    Object.keys(books).forEach((ID) => {
+      const booksTemp = {};
+      booksTemp.id = ID;
+      booksTemp.title = books[ID][0].title;
+      booksTemp.category = books[ID][0].category;
+      dispatch(addBook(booksTemp));
+    });
+  });
+};
+
 const reducer = (state = initialState, action) => {
   switch (action.type) {
     case ADD_BOOK:
+      newBook(action.payload);
       return [...state, action.payload];
 
     case REMOVE_BOOK:
+
+      deleteBook(action.payload);
       return state.filter((book) => book.id !== action.payload);
 
     default:
```

Comment:
- [ ] There are some console errors that appear in your project. The video explains those so please watch it. 😉 

![image](https://user-images.githubusercontent.com/79876959/145360195-5169a0b9-0ad4-4203-8639-a714a7bd91f4.png)

![image](https://user-images.githubusercontent.com/79876959/145360230-38743034-2c02-453c-8759-69fc3432a4d6.png)


https://user-images.githubusercontent.com/79876959/145360677-f26acc9e-deeb-491e-9c89-1c3003c66915.mp4

Reply:
Hello @ArthurGC, this is the best CR I have ever got, I really wanted to thank you for the time you took in reviewing my code and making this video, it was really helpful!!